nodejs/awsx/apigateway/api.ts

```diff
@@ -57,6 +57,11 @@ export type Response = awslambda.APIGatewayProxyResult;
  */
 export type Route = EventHandlerRoute | StaticRoute | IntegrationRoute | RawDataRoute;
 
+/**
+ * Subset of `Route` types that can be passed in in `Output` form to the API.  These Route types
```

Comment:
This stutters `that can be passed in in`.  Maybe `passed in as the` ... 🤔

Reply:
Fixed!